static/styles/rendered_markdown.css

```diff
@@ -5,6 +5,16 @@
         margin: 3px 0;
     }
 
+    /* Paragraphs: Ensure that the first paragraph and last paragraph
+    don't have blank area at the top and bottom respectively. */
+    p:first-child {
+        margin-top: 0;
+    }
+
+    p:last-child {
+        margin-bottom: 0;
+    }
```

Comment:
It seems like these CSS changes might affect the actual rendering of messages, which we don't want to do. Do we instead want to add these inside a `#preview_area .rendered_markdown` type of block?  That could work well, though I'll note that I `preview_area` is probably not the name of the element.